src/internal/fungible_asset.ts

```diff
@@ -54,3 +59,47 @@ export async function getFungibleAssetMetadata(args: {
 
   return data.fungible_asset_metadata;
 }
+
+export async function getFungibleAssetActivities(args: {
+  aptosConfig: AptosConfig;
+  options?: {
+    pagination?: PaginationArgs;
+    whereCondition?: {
```

Comment:
not completely sure why we need to define what user can filter by? where condition should accept user customized properties, like https://github.com/aptos-labs/aptos-ts-sdk/blob/main/src/api/event.ts#L75

Reply:
ohh okay, didn't realize you can assign the bool exp like that. 

Will update